### PR TITLE
Fix typo in rule name in docs

### PR DESCRIPTION
**Pull request by denis-sokolov**
Seems to be a historic typo in the docs.

### CODE REVIEW

**Review by sloops77 — APPROVED** [review]

**Comment by sloops77** [comment]
@denis-sokolov thanks!

**Comment by denis-sokolov** [comment]
@sloops77, you probably want to press the merge button. I can’t merge this myself even if you approve, I’m an outside contributor.

**Pull request merged**
